### PR TITLE
Move all runtime tests to polkadot-sdk

### PR DESCRIPTION
**Pull request by yrong**
https://github.com/Snowfork/polkadot-sdk/pull/92

### CODE REVIEW

**Review by claravanstaden — APPROVED** [review]
Great, thanks! 🏅

**Review by alistair-singh — APPROVED** [review]
Nice!

**Review by vgeddes — COMMENTED** [review]
I disagree - see my full comment on slack: https://snowfork.slack.com/archives/G01BGMGLAC9/p1704789482331339?thread_ts=1704782841.923229&cid=G01BGMGLAC9

The support code in `parachain/runtime/test_cases.rs` should remain in our repo

File: parachain/runtime/tests/Cargo.toml
Comment (vgeddes) [inline parachain/runtime/tests/Cargo.toml:1]: Given its new purpose, we should rename this crate to `snowbridge-runtime-test-common`.

The directory should also be renamed accordingly: `parachain/runtime/test-common`

**Review by vgeddes — COMMENTED** [review]
Looks better. Just need to rename the crate.

**Review by vgeddes — APPROVED** [review]
+1

**Pull request merged**